File: utils/api/dtos/auditoria.go
package dtos

import "time"

type DocumentAudit struct {
	ID          int        `json:"id_formulario"`
	Informacion string     `json:"informacion"`
	Nombre      string     `json:"nombre"`
	Version     string     `json:"version"`
	Fecha       *time.Time `json:"fecha"`
}

File: static/js/auditoria.js
$(document).ready(function() {

   
    Paginar("auditoria");
  });


  async function getFormularios() {
    const request = await fetch("http://34.192.187.56:8080/formulario", {
      method: 'GET',
    })
  
    let response = await request.json()
    console.log(response)
    if (request.ok) {
              let text = ``
              response.forEach(auditoria => {
                  text +=
                      `
                       <tr>
                          <th>${auditoria.nombre}</th>
                          <th>${auditoria.informacion}</th>
                          <th>${auditoria.version}</th>
                          <th>${auditoria.fecha}</th>
                          
                      </tr>
                      `
              });
              document.getElementById("auditoriaTBody").innerHTML = text;
    }
    
  }